index.bs

```diff
@@ -505,9 +521,35 @@ The `:picture-in-picture` <a>pseudo-class</a> MUST match any <a>element</a>
 
 <em>This section is non-normative.</em>
 
-The API applies only to {{HTMLVideoElement}} in order to start on a minimal
-viable product that has limited security issues. Later versions of this
-specification may allow PIP-ing arbitrary HTML content.
+The {{HTMLVideoElement}} Picture-in-Picture API has limited security issues.
+
+## Restrictions on arbitrary content ## {#arbitrary-content-restrictions}
+
+The <a>Picture-in-Picture browsing context</a> MUST have top-level navigation
+blocked and the following <a>sandboxing flag set</a> applied:
+- <a>sandbox propagates to auxiliary browsing contexts flag</a>
+- <a>sandboxed auxiliary navigation browsing context flag</a>
+- <a>sandboxed document.domain browsing context flag</a>
+- <a>sandboxed forms browsing context flag</a>
+- <a>sandboxed modals flag</a>
+- <a>sandboxed navigation browsing context flag</a>
+- <a>sandboxed orientation lock browsing context flag</a>
+- <a>sandboxed pointer lock browsing context flag</a>
+- <a>sandboxed presentation browsing context flag</a>
+
+If the user agent restricts autoplay then this SHOULD be allowed by default.
```

Comment:
Can you link to autoplay in the HTML spec?

Reply:
Done.